core/storage/collection/gae_models.py

```diff
@@ -615,6 +615,9 @@ class CollectionSummaryModel(base_models.BaseModel):
     viewer_ids = ndb.StringProperty(indexed=True, repeated=True)
     # The user_ids of users who have contributed (humans who have made a
     # positive (not just a revert) change to the collection's content).
+    # NOTE TO DEVELOPERS: contributor_ids and contributors_summary need to be
```

Comment:
Just to check -- is this validated in prod_validation_jobs_one_off.py? If not, it probably should be...

/cc @ankita240796

Reply:
It is checked!

Reply:
Great, thanks for confirming! Merged :)